### DISCUSSION TITLE
Welcome to PHP-SQL-Chat Discussions!

### DISCUSSION THREAD

**Discussion by Maxhu787** [g4o2/PHP-SQL-Chat#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Immortality-IMT** [comment]
Hi, I am looking a simple chatroom, so I can contribute to development. Where is the SQL file, so I can setup locally. 

Thanks
Tony

**Comment by Maxhu787** [comment]
## Glad to know that you are interested
Here is the sql export
I'm currently working on a making it a proper real time chat
[g4o2-chat-2022-12-4.zip](https://github.com/g4o2/PHP-SQL-Chat/files/10148307/g4o2-chat-2022-12-4.zip)
